lib/handlers/get.js

```diff
@@ -135,7 +135,7 @@ async function globHandler (req, res, next) {
   const ldp = req.app.locals.ldp
   // TODO: This is a hack, that does not check if the target file exists, as this is quite complex with globbing.
   // TODO: Proper support for this is not implemented, as globbing support might be removed in the future.
-  const filename = ldp.resourceMapper.getFullPath(req)
+  const filename = ldp.resourceMapper.getFilePath(req)
   const requestUri = (await ldp.resourceMapper.mapFileToUrl({ path: filename, hostname: req.hostname })).url
```

Comment:
So what is happening above seems to be answering the question “what is the full request URL”? Moreover, that question seems to be answered multiple times. Let's create a method for it.